curriculum/challenges/chinese/01-responsive-web-design/applied-visual-design/use-the-u-tag-to-underline-text.md

```diff
@@ -25,7 +25,7 @@ dashedName: use-the-u-tag-to-underline-text
 assert($('u').length === 1);
 ```
 
-`u` 标签的文本内容应为 `Ph.D. students`。
+`u` 标签的文本内容应为 `Ph.D.  students`。
```

Comment:
```suggestion
`u` 标签的文本内容应为 `Ph.D. students`。
```

Reply:
I think I added that extra space, even though on Crowdin it did not appear as if I was.